### PR TITLE
Extract napari to napari-ome-zarr

### PR DESCRIPTION
**Pull request by joshmoore**
seealso: https://github.com/ome/napari-ome-zarr/pull/2

### CODE REVIEW

**Review by sbesson — COMMENTED** [review]
As mentioned before, my inclination in this case would rather be to separate the napari/client layer into its separate downstream module as proposed here. The decision boils down the classical trade-off  debate between monolithic vs decoupled repos.

Usually one of the major downside of component decoupling appears when we start working into API/specification changes at the core library level and test these changes in the client. For developers, I assume `pip install napari-ome-zarr` followed by `pip install -e .` from a working copy of this repository would do the trick. Is this also sufficient for testing purposes? Do we foresee the need for coupled CI builds at this stage?

The only other aspect I noticed is that GitHub actions still includes a large number of matrix builds. I assume that `PyQt5` and `PySide2` can be removed from `tox.ini` and the testing matrix?

File: README.rst [hunk 1]
```diff
@@ -61,30 +61,6 @@ To download all the resolutions and metadata for an image::
     # Specify output directory
     $ ome_zarr download https://s3.embassy.ebi.ac.uk/idr/zarr/v0.1/6001240.zarr/ --output image_dir
 
-napari plugin
-=============
-
-Napari will use `ome-zarr` to open images that the plugin recognises as ome-zarr.
```
[the remark this answers is not in the document]
Comment (sbesson) [reply README.rst:67]: Scratch that, I just opened https://github.com/ome/napari-ome-zarr/pull/2

**Review by sbesson — APPROVED** [review]
@joshmoore I would propose to get this merged so that the `main` branch reflects the extraction of the `napari` plugin and look into a few actions as a follow-up including:

- GitHub actions cleanup (https://github.com/ome/ome-zarr-py/pull/86#pullrequestreview-671220296)
- dependency review & cleanup (https://github.com/ome/ome-zarr-py/issues/90)

**Comment by joshmoore** [comment]
@sbesson : agreed. (I actually thought it had been merged. Got lost in the meeting fun)

Any thought on a version? 0.1.0 is still open to us.

**Comment by joshmoore** [comment]
> * GitHub actions cleanup ([#86 (review)](https://github.com/ome/ome-zarr-py/pull/86#pullrequestreview-671220296))

I'll open a PR shortly.

**Pull request merged**